cs3api4lab/api/cs3_file_api.py

```diff
@@ -210,7 +210,8 @@ def move(self, source_path, destination_path, endpoint=None):
         src_reference = FileUtils.get_reference(source_path, endpoint)
         dest_reference = FileUtils.get_reference(destination_path, endpoint)
 
-        if self.storage_logic.stat(destination_path, endpoint):
+        stat = self.storage_api.stat(destination_path, endpoint)
+        if stat.status.code == cs3code.CODE_OK:
```

Comment:
Can it be that is not the file that exists, but the call failed?

Reply:
not sure what you mean here, if the call fails then subsequent calls will also probably fail

Reply:
I was wondering if it's possible that the call failed for some reason but the file actually exists. The next call will do a move on top of a file.
But, as you say, if it's not clear from the error code wether the file does not exist or smth else failed, I guess we should continue...

Reply:
This is the same question as above